modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2850,11 +2851,7 @@ private void _validateOneToOneUpdate(
 	}
 
 	private Map<String, Serializable> _validatePicklistDefaultValue(
```

Comment:
Hi @pedro-oliveira446 could you please squase this commit with the previous one?